v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/PipelineController.java

```diff
@@ -265,7 +267,7 @@ static ISchemaMapper getSchemaMapper(SourceDbToSpannerOptions options, Ddl ddl)
   static List<String> listTablesToMigrate(String tableList, ISchemaMapper mapper, Ddl ddl) {
     List<String> tablesFromOptions =
         StringUtils.isNotBlank(tableList)
-            ? Arrays.stream(tableList.split(",")).collect(Collectors.toList())
+            ? Arrays.stream(tableList.split("\\||,")).collect(Collectors.toList())
```

Comment:
Could we use some other separator?
Bash would make the user quote the pipe while issuing the gcloud cli.

Reply:
Updated to colon